lib/proxy/log.rb

```diff
@@ -14,19 +14,13 @@
 
 module Proxy
   module Log
-    @@logger = nil
-
     def logger
```

Comment:
ATM this is the main interface to logger, a warning on every call would be quite annoying. I haven't given up on switching to events, at which point this would go away...